File: ITManagementServer/src/main/java/models/requestModels/requests/CreateRequestRequestModel.java
package models.requestModels.requests;

public class CreateRequestRequestModel {
    private int WorkerId;
    private String RequestDescription;
    private int RequestType;

    public int getWorkerId() {
        return WorkerId;
    }

    public void setWorkerId(int workerId) {
        WorkerId = workerId;
    }

    public String getRequestDescription() {
        return RequestDescription;
    }

    public void setRequestDescription(String requestDescription) {
        RequestDescription = requestDescription;
    }

    public int getRequestType() {
        return RequestType;
    }

    public void setRequestType(int requestType) {
        RequestType = requestType;
    }
}

File: ITManagementClient/Models/RequestModels/Requests/CreateRequestRequestModel.cs
using ITManagementClient.Models.Enums;

namespace ITManagementClient.Models.RequestModels.Requests
{
    public class CreateRequestRequestModel
    {
        public int WorkerId { get; set; }

        public string RequestDescription { get; set; }

        public RequestType RequestType { get; set; }
    }
}